applications/asset_tracker_v2/pm_static_nrf9160dk_nrf9160_ns_0_14_0.yml

```diff
@@ -0,0 +1,5 @@
+mcuboot_secondary:
+  region: external_flash
+  device: DT_CHOSEN(nordic_pm_ext_flash)
```

Comment:
Is the device line necessary? I thought this was inherited from the external_flash region

Reply:
You're right! Just confirmed by doing an mcuboot update without the `device` line, and the new version persisted. Have pushed an update removing this line now.